### DISCUSSION TITLE
How to get spatial correlations?

### DISCUSSION THREAD

**Discussion by FrancoAquistapace** [killiansheriff/ChemicalMotifIdentifier#3] [Q&A]
Hi there! I have been trying to use the code for my own research. I wanted to kindly ask if someone could explain how to get the spatial correlation function and characteristic length-scale of chemical fluctuations, as shown in Figure 4 of the publication "Quantifying chemical short-range order in metallic alloys". Thanks in advance!

**Comment by killiansheriff** [comment]
Hi, 

Sorry for missing this! 

I have included in this [tutorial](https://github.com/killiansheriff/ChemicalMotifIdentifier/blob/main/examples/doi-10.48550-arXiv.2311.01545/fcc_ternary_system.ipynb) an example on how to get the dissimilarity between two motifs. SI Section 7 of the paper gives a detailed description of how to compute the correlation length scales (the code can be found [here](https://github.com/killiansheriff/ChemicalMotifIdentifier/blob/main/chemicalmotifidentifier/_src/correlations.py)). 

Let me know if you need more details,

Hope this helps! 

Best, 
Killian
